packages/react-bootstrap-table2-example/stories/index.js

```diff
@@ -94,4 +95,5 @@ storiesOf('Cell Editing', module)
   .add('Row Level Editable', () => <RowLevelEditableTable />)
   .add('Column Level Editable', () => <ColumnLevelEditableTable />)
   .add('Rich Hook Functions', () => <CellEditHooks />)
-  .add('Validation', () => <CellEditValidator />);
+  .add('Validation', () => <CellEditValidator />)
+  .add('Editable as callback', () => <ColumnEditableAsCallback />);
```

Comment:
Please name the title as Cell Level Editable and move to the below of 'Column Level Editable'. thanks !!